packages/common/src/controllers/AccountController.ts

```diff
@@ -395,7 +395,7 @@ export default class AccountController {
 
     // The subscription data takes a few seconds to load after it's purchased,
     // so here's a delay mechanism to give it time to process
-    if (delay > 0) {
+    if (delay && delay > 0) {
```

Comment:
this doesn't make a difference, right? Or maybe for typescript it does...

Reply:
Yes, it's not allowed to use the `>` operator on `undefined`